packages/augur-ui/src/modules/common/icons.tsx

```diff
@@ -2860,6 +2860,17 @@ export const CreatedMarketsIcon = (
   </svg>
 );
 
+export const FingersCrossed = (
+  <svg viewBox="0 0 24 24" fill="none">
+    <path d="M17.0435 1.59842C16.8597 1.5268 16.6635 1.49215 16.4663 1.49645C16.269 1.50075 16.0746 1.54392 15.8941 1.62348C15.7135 1.70304 15.5505 1.81743 15.4142 1.9601C15.278 2.10277 15.1712 2.27092 15.1 2.45492L12.363 9.49842H10C9.6791 9.49802 9.36528 9.59283 9.09828 9.77083C8.83127 9.94884 8.62306 10.202 8.5 10.4984H7.5C7.10218 10.4984 6.72064 10.6565 6.43934 10.9378C6.15804 11.2191 6 11.6006 6 11.9984V17.4984C6.0009 18.1599 6.18926 18.8077 6.54324 19.3665C6.89721 19.9253 7.40232 20.3724 8 20.6559V22.9984C8 23.131 8.05268 23.2582 8.14645 23.352C8.24021 23.4457 8.36739 23.4984 8.5 23.4984H15.5C15.6326 23.4984 15.7598 23.4457 15.8536 23.352C15.9473 23.2582 16 23.131 16 22.9984V20.3664C16.4627 20.0447 16.8408 19.6159 17.1019 19.1165C17.3631 18.6171 17.4997 18.062 17.5 17.4984V11.9984L15.08 10.7884L17.9 3.54192C17.9716 3.35809 18.0063 3.16194 18.002 2.96471C17.9977 2.76747 17.9545 2.57301 17.8749 2.39248C17.7954 2.21195 17.681 2.04889 17.5383 1.91263C17.3956 1.77637 17.2275 1.6696 17.0435 1.59842Z" fill="#FFD764"/>
```

Comment:
we ok with providing ``` fill="#E2AC4B"``` on the path directly?

Reply:
for now, yes, only because the colors used on this particular icon aren't shared anywhere else and won't be changing based on theme. I'm hoping for a reason to make it common instead but if this is a one off icon/color then we can let it slide.